### PR TITLE
Blog: Updates 2022-08

### PR DESCRIPTION
**Pull request by psiinon**
Planned to be published on 31th Aug, so will be updated for any more changes that happen before then.
I've still got more updated add-ons to add.
Does the format / style look ok?
Any suggestions for improvements??

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Comment by psiinon** [comment]
And have I missed anything??

File: site/content/blog/2022-08-31-zap-updates-2022-august/index.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+ZAP is run against a set of test applications every day - for more details see the [Test Scans](/docs/scans/) page.
+
+This month [OWASP Benchmark](https://owasp.org/www-project-benchmark/) was added to that list.
+Initially we are just testing ZAP vs the Command Injection vulnerabilities but we will be expanding support to cover 
+all of the relevant tests a.s.a.p.
+
+For more details see [ZAP vs OWASP Benchmark](/docs/scans/benchmark/).
+
+#### HUD problems on Firefox
```
Comment (kingthorin) [inline site/content/blog/2022-08-31-zap-updates-2022-august/index.md:60]:
```suggestion
#### HUD Problems on Firefox
```

File: site/content/blog/2022-08-31-zap-updates-2022-august/index.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+Unfortunately the [ZAP HUD](https://github.com/zaproxy/zap-hud/) is no longer working with the latest version of Firefox.
```
Comment (kingthorin) [inline site/content/blog/2022-08-31-zap-updates-2022-august/index.md:62]:
```suggestion
Unfortunately the [ZAP HUD](https://github.com/zaproxy/zap-hud/) is no longer working with the latest versions of Firefox.
```

**Comment by psiinon** [comment]
Updated for stats, supporters and a few other things...

File: site/content/blog/2022-08-31-zap-updates-2022-august/index.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+
+### Ongoing Work
+
+#### Networking
+
+Work to upgrade the ZAP networking stack to support modern protocols is ongoing.
+The next phase will add initial support for HTTP/2 - this is a significant change and we expect this to be merged after
+2.12.0 has been released.
+
+#### Spider Migration to an Add-on
```
Comment (ricekot) [inline site/content/blog/2022-08-31-zap-updates-2022-august/index.md:55]: The content under this section could include a link to the spider blog post once that's published.

**Comment by psiinon** [comment]
OK, I think this is now ready to publish :)
Anything missing / wrong?
Should we publish it today or tomorrow?

**Review by kingthorin — APPROVED** [review]

**Comment by psiinon** [comment]
Added image c/o @ricekot

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**